### ISSUE TITLE
Switch corresponding results pane when switching query tabs

### ISSUE THREAD

**Issue by Tyler-Oreskey** [microsoft/vscode-mssql#1480]
- MSSQL Extension Version: 1.7.0
- VSCode Version: 1.40.0
- OS Version: Catalina 10.15.1

Description:
As a user I would like for SQL query panes and their corresponding results to stay paired when changing panes.

Steps to reproduce:

1. Create and run at least two SQL queries in different query panes.
2. Switch back and forth between the query panes.
3. Notice the results do not switch with their corresponding pane.

**Comment by Kenobi-the-2nd** [comment]
Agreed, this is definitely needed to improve workflow. Another issue [here ](https://github.com/microsoft/vscode-mssql/issues/1587)mentions the same problem.

**Comment by NicholasTuttle** [comment]
I would certainly like to see this as well. It's one of the biggest reasons I use SSMS over VS Code.

**Comment by kburtram** [comment]
The results pane experience has been completely redesigned since this suggestion was filed, and within the tool windows pane the results grid does track the active editor.  Please log a new issue relate against the current experience if there are further suggestions.  Thanks for logging this issue!
